src/plugins/condition/ConditionManager.js

```diff
@@ -322,8 +322,11 @@ export default class ConditionManager extends EventEmitter {
         let timestamp = {};
         timestamp[timeSystemKey] = normalizedDatum[timeSystemKey];
 
-        this.conditions.forEach(condition => {
+        //We want to stop when the first condition evaluates to true.
+        this.conditions.some((condition) => {
```

Comment:
you can return `result` from `getResult` in `openmct/src/plugins/condition/Condition.js`, then 
`this.conditions.some(condition =>  condition.getResult(normalizedDatum));`

Reply:
Yeah, it's sort of the same thing. I think this is less of a change to the files. I'd prefer to leave it this way.

Reply:
hmm, found it confusing because of name `getResult` and its not returning any result.

Reply:
I've changed the method name to updateResult in Condition and Criterion classes.